tests/setup-softokn.sh

```diff
@@ -155,11 +156,12 @@ export PKCS11_PROVIDER_DEBUG="file:${BASEDIR}/${TMPPDIR}/p11prov-debug.log"
 export PKCS11_PROVIDER_MODULE="${SOFTOKNPATH}/libsoftokn3.so"
 export OPENSSL_CONF="${OPENSSL_CONF}"
 export TESTSDIR="${TESTSDIR}"
+export TESTBLDDIR="${TESTBLDDIR}"
 
 export TOKDIR="${BASEDIR}/${TOKDIR}"
 export TMPPDIR="${BASEDIR}/${TMPPDIR}"
 export PINVALUE="${PINVALUE}"
-export PINFILE="${BASEDIR}/${PINFILE}"
+export PINFILE="${PINFILE}"
```

Comment:
I do not see how changing this PINFILE export makes a difference or is correct.
If you change the openssl.cnf var to use testsblddir then you should also export this as `${TESTSBLDDIR}/${PINFILE}` I think and same for TMPDIR and TOKDIR.

Reply:
Actually looking at tests we do not need to export PINFILE at all, nothing but openssl.cnf references it anyway. And I checked that it's path is now correct in testvars, but it is unesed, so would you mind just removing it ?
Also you did not correct it in setup-softhsm.sh, just remove it from exports there too

Reply:
OK, will remove "export PINFILE=..."

Reply:
done